Assets/Scripts/Controllers/PlayerInputController.cs

```diff
@@ -20,4 +20,9 @@ public override bool RetrieveJumpButtonReleased()
     {
         return Input.GetButtonUp("Jump");
     }
+
+    public override bool RetrieveDashInput()
+    {
+        return Input.GetButtonDown("Dash");
+    }
```

Comment:
We need to use the MInput class for new input. We'll have to specify the exact KeyCode as well as the GamepadCode binding we want to use.

On a sidenote, I'm not sure why this file is showing the old `Input` usage and not `MInput`. That's weird

Reply:
The input changes aren't on this branch. I figured it would be easier to merge this into development and then do a quick update to fix the inputs